Dockerfile

```diff
@@ -0,0 +1,44 @@
+################################################################################################################ builder
+
+FROM maven:3-jdk-8 as builder
+WORKDIR /build
+
+# this block will make docker cache the dependencies for subsequent builds of the same image.
+COPY pom.xml .
+COPY subsonic-rest-api/pom.xml ./subsonic-rest-api/pom.xml
+COPY integration-test/pom.xml ./integration-test/pom.xml
+COPY airsonic-sonos-api/pom.xml ./airsonic-sonos-api/pom.xml
+COPY airsonic-main/pom.xml ./airsonic-main/pom.xml
+RUN mvn de.qaware.maven:go-offline-maven-plugin:resolve-dependencies
+
+COPY . .
+RUN mvn package
+
+################################################################################################################ runtime
+
+FROM alpine:3.9 as runtime
```

Comment:
I'd just take the openjdk's jre-alpine base instead of taking an alpine base and installing openjdk's jre into it

Reply:
I'd also suggest using [adoptopenjdk/openjdk8-openj9](https://hub.docker.com/r/adoptopenjdk/openjdk8-openj9/).

- https://github.com/docker-library/official-images/pull/5710#issuecomment-483219593
- https://www.eclipse.org/openj9/oj9_performance.html

Reply:
agree with > JDK8, and I am familiar with adoptopenjdk. Perhaps we could go all the way to JDK11 since it has LTS?

Reply:
At some point in time we might have to convince the maintainers here to move to 11, but I don't know if that's something they'd be amenable to in this pr

Reply:
Did we want to use the [`adoptopenjdk/openjdk8-openj9`](https://hub.docker.com/r/adoptopenjdk/openjdk8-openj9/) image? At least it stayed on Java 8. OpenJ9 is a faster, reliable, OSS alternative to Oracle's JVM.

Reply:
I did run some tests, `adoptopenjdk/openjdk9:alpine` requires workarounds for https://github.com/AdoptOpenJDK/openjdk-docker/issues/75, `adoptopenjdk/openjdk11:alpine` and `adoptopenjdk/openjdk8-openj9:alpine` work fine. 

Going to use `adoptopenjdk/openjdk8-openj9:alpine` in this PR and run more tests with adoptopenjdk/openjdk11:alpine on my personal instance.